### ISSUE TITLE
Tab Stash button disappeared so saved tabs were lost

### ISSUE THREAD

**Issue by groovybluedog** [josh-berry/tab-stash#59]
I saved a bunch of tabs with this plugin, and when I next closed and opened Firefox, the button on the toolbar is now a white space where the icon should be.

![image](https://user-images.githubusercontent.com/16142383/82384920-d324e000-9a28-11ea-9915-0ac7288cc2eb.png)

When I click in this white space, nothing loads and I am unable to retrieve my saved tabs.

The reviews and developer response among with other issues are positive so this is probably either an incompatability with one of my other addons or just a generic fault for me, but fustrating still.

Other extensions in use are base 64 decoder, Clear Cache, Enhancer for YouTube, Keepa Amazon price tracker, LastPass, Pushullet, Reddit Hide Promoted and Winscribe VPN.

Thank you.

**Comment by josh-berry** [comment]
hm, that definitely shouldn't be happening. What version of Firefox are you using, on what OS?

Whatever tabs you have saved should still be there, so in the meantime, you can always find things again through your Firefox bookmarks (menu > Library > Bookmarks > Show All Bookmarks).

Since even the icon itself seems to be missing, the most likely explanation is that somehow Firefox didn't install the extension correctly—if you haven't already, I suggest uninstalling and reinstalling it, and see if it sticks around after restarting your browser. (Since your tabs are saved as bookmarks, they will be saved even if you uninstall/reinstall. You will need to reset any Tab Stash options you had changed, but your tabs themselves will be fine.)

**Comment by josh-berry** [comment]
Hi, did uninstalling and reinstalling solve your problem?

**Comment by groovybluedog** [comment]
Thanks for your reply and apologies for not getting in touch sooner. I was using Firefox 76 (or whatever the most recent version of Firefox was at that time), on Windows 10. Thankfully, I was able to retrieve the bookmarks from Firefox, and I reinstalled the addon which seemed to work, howeve, with all due respect to the effort you have put in to your addon and responding to GitHub, I personally could not get on the interface and functionality and decided to uninstall. :)

**Comment by josh-berry** [comment]
Fair enough, glad you were able to resolve the issue. I do understand—there are a lot of ways to manage tabs and it's important to find what works for you. Tab Stash is not for everyone and that's okay! Best wishes.
